ci/prow/e2e-crd-variable

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -x
```

Comment:
```suggestion
set -xe
```

Reply:
I think that it we use -e here it will exit before getting to the troubleshoot if any of the previous commands fail, as @qbarrand commented [here](https://github.com/kubernetes-sigs/kernel-module-management/pull/166#discussion_r1033489465)

Reply:
If we move the troubleshooting back to the Action YAML, `set -euxo pipefail` makes sense IMHO.